airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeSqlOperations.java

```diff
@@ -65,9 +90,18 @@ public void insertRecordsInternal(final JdbcDatabase database,
     // FROM VALUES
     // (?, ?, ?),
     // ...
-    final String insertQuery = String.format(
-        "INSERT INTO %s.%s (%s, %s, %s) SELECT column1, parse_json(column2), column3 FROM VALUES\n",
-        schemaName, tableName, JavaBaseConstants.COLUMN_NAME_AB_ID, JavaBaseConstants.COLUMN_NAME_DATA, JavaBaseConstants.COLUMN_NAME_EMITTED_AT);
+    final String insertQuery;
+    if (use1s1t) {
+      // Note that the column order is weird here - that's intentional, to avoid needing to change
+      // SqlOperationsUtils.insertRawRecordsInSingleQuery to support a different column order.
+      insertQuery = String.format(
+          "INSERT INTO %s.%s (%s, %s, %s) SELECT column1, parse_json(column2), column3 FROM VALUES\n",
+          schemaName, tableName, JavaBaseConstants.COLUMN_NAME_AB_RAW_ID, JavaBaseConstants.COLUMN_NAME_DATA, JavaBaseConstants.COLUMN_NAME_AB_EXTRACTED_AT);
```

Comment:
What is "`VALUES`" here?  The tmp table from the CSV we just uploaded?

Reply:
this is `insertRecordsInternal`, which I think was used by the (now-defunct) standard inserts mode. It's just a typical sql insert statement

I'm pretty sure this method is only used from tests, but it's required by the `JdbcSqlOperations` interface, and there's enough references around the inheritance tree that I'm not super confident in just killing it right now.

Reply:
got it.  Write an issue to rip it out then please!

Reply:
https://github.com/airbytehq/airbyte/issues/28648